dev/reference/transactions/transaction-pessimistic.md

```diff
@@ -48,29 +52,16 @@ TiDB 默认使用乐观事务模式，存在事务提交时因为冲突而失败
 
 如果想要禁用悲观事务特性，可以修改配置文件，在 `[pessimistic-txn]` 类别下添加 `enable = false`.
 
-## 相关配置参数
-
-相关配置都在 `[pessimistic-txn]` 类别下，除了前面介绍过的 `enable`，还可配置以下参数：
-
-- `ttl`
+## 和 MySQL innoDB 的差异
 
-    ```
-    ttl = "40s"
-    ```
+- TiDB 在使用范围做为 WHERE 条件，执行 DML 和 SELECT FOR UPDATE 语句时不会阻塞范围内并发的 INSERT 语句的执行。 
 
-    `ttl` 是悲观事务的锁超时时间，默认值 40 秒，可以在 15 秒到 120 秒之间配置，超出这个范围会报错。如果事务的执行时间超过了 `ttl`，事务会失败。设置过大，会存在 `tidb-server` 宕机时，残留的悲观锁长时间阻塞写的问题，设置的过小，会存在事务来不及执行完，就被其他事务 rollback 的问题。
+innoDB 通过实现 gap lock，支持了阻塞范围内并发的 INSERT 语句的执行，主要目的是为了支持 statement based binlog。
 
-- `max-retry-count`
+有些业务通过降低隔离级别到 READ COMMITTED 来避免 gap lock 导致的并发性能问题。
 
-    ```
-    max-retry-count = 256
-    ```
-
-    悲观事务会在事务内自动重试单个语句，这个配置指定单个语句最大重试次数，设置这个参数是为了避免极端情况下，出现无限重试，无法停止。
-正常情况下不需要修改这个配置。
-
-## 已知的局限
-
-- 不支持 GAP Lock 和 Next Key Lock 在悲观事务内通过范围条件来更新多行数据的时候，其他的事务可以在这个范围内插入数据而不会被阻塞。
+TiDB 不支持 gap lock，也就不需要付出相应的并发性能的代价。
```

Comment:
```suggestion
innoDB 通过实现 gap lock，支持阻塞 range 内并发的 INSERT 语句的执行，其主要目的是为了支持 statement based binlog，因此有些业务会通过降低隔离级别到 READ COMMITTED 来避免 gap lock 导致的并发性能问题。TiDB 不支持 gap lock，也就不需要付出相应的并发性能的代价。
```

Reply:
Combine the three sentences into one paragraph and add some conjunctions to make the paragraph cohesive.